.gitignore

```diff
@@ -9,6 +9,10 @@ package-lock.json
 # tarball from `yarn pack`
 *.tgz
 
+# docs, build files, etc.
 coverage
 dist
 docs
+
+# misc
+.python-version
```

Comment:
What's this for?

Reply:
Leftovers from the pre-`yarn` days

I have `pyenv` locally and guess what, `scrypt` didn't build with `3.x`

Reply:
But I deleted it